packages/connectkit/src/wallets/walletConfigs.tsx

```diff
@@ -212,8 +225,8 @@ export const walletConfigs: {
       return isAndroid()
         ? uri
         : `https://rnbwapp.com/wc?uri=${encodeURIComponent(
-            uri
-          )}&connector=connectkit`;
+          uri
+        )}&connector=connectkit`;
```

Comment:
Can we remove these extra formatting changes and update with latest `main` branch @MarkNerdi?